### DISCUSSION TITLE
Add Multi Language option

### DISCUSSION THREAD

**Discussion by leranp** [OHF-Voice/intents#1196] [_Ideas]
Hi,

My default language in HA is English, but my native language is Hebrew, and i can't use the commands ,is there any plans to add an option to change it in the configuration file?

**Comment by FearcharM** [comment]
This is clearly not an isolated request: most of the globe's population is multilingual (at the very least bilingual).

**Reply by tetele** [reply]
The initial discussion is more than 1.5 years old. What exactly can't you do with Assist atm?
